include/mgp.hpp

```diff
@@ -2725,6 +2731,17 @@ inline void Node::SetProperty(std::string property, Value value) {
   mgp::vertex_set_property(ptr_, property.data(), value.ptr());
 }
 
+inline void Node::SetProperties(std::unordered_map<std::string_view, Value> properties) {
```

Comment:
What if this function accepts `mgp::Map` since we are already transforming from unordered_map to mgp_map which is std::map? So that would be one less transformation and less memory?

Reply:
either way you have to construct the mgp::Map which means it happens either in the API or in the query module. I'd rather want users to work with STL because we can't guess what is the input to the API (map as argument from query module or anything else)